packages/client/src/pages/IntroPage.tsx

```diff
@@ -23,12 +25,10 @@ const IntroPage: React.FC = () => {
   ) as IntroProps;
 
   const goToNext = () => {
-    // @TODO: Change and refactor this because the next step is not always LOCATION_INPUT
-    matomoTrackEvent({
-      action: actions.ACTIVE_STEP,
-      name: sections.LOCATION_INPUT,
-    });
-
+    // Unset the current checker in case the topicData contains an old checker
```

Comment:
Is it correct the matomo event disappeared here?

Reply:
If you look closely you can see the event has been moved to `packages/client/src/pages/CheckerPage.tsx`